File: igor/Problema0041/problema0041.java
import java.util.Scanner;
public class problema0041 {

    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("escreve o dividendo: ");
        int x = input.nextInt();
        System.out.println("escreve o divisor: ");
        int y = input.nextInt();

        int z = x / y;
        int w = x % y;

        System.out.println("Dividendo: "+x);
        System.out.println("Divisor: "+y);
        System.out.println("Quociente: "+z);
        System.out.println("Resto: "+w);
    }
}

File: igor/Problema0041/problema0041.js
const prompt = require("prompt-sync")({ sigint: true });

var x = prompt("Escreve o dividendo: ")
var y = prompt("Escreve o divisor: ")

var z = x/y
var w = x%y
var t = parseInt(z)

console.log("Dividendo: "+x)
console.log("Divisor: "+y)
console.log("Quociente: "+t)
console.log("Resto: "+w)